cerberus/kubernetes/client.py

```diff
@@ -109,3 +109,36 @@ def monitor_component(iteration, component_namespace):
     logging.info("Iteration %s: %s: %s"
                  % (iteration, component_namespace, watch_component_status))
     return watch_component_status, failed_component_pods
+
+
+def monitor_cluster_operator(iteration, cluster_operators):
+
+    failed_operators = []
+    for operator in cluster_operators['items']:
+
+        #loop through the conditions in the status section to find the dedgraded condition
+        for status_cond in operator['status']['conditions']:
+            if status_cond['type'] ==  "Degraded":
+                
+                logging.info("Iteration %s: Operator %s: %s" % (iteration,operator['metadata']['name'], status_cond['status']))
+                #if the degraded status is true, add it to the failed operators to return
+                if status_cond['status'] != "False":
+                    failed_operators.append(operator['metadata']['name'])
+                break
+
+    #if failed operators is not 0, return a failure
+    #else return pass
+    if len(failed_operators) != 0:
+        status = False
+    else:
+        status = True
+
+    return status, failed_operators
+
+
+def monitor_cluster_operators(iteration, cluster_operators):
```

Comment:
+1 to merging this function with monitor_cluster_operator function like you mentioned in the commit message.